webapp/webapp/etc/cron.d/db-backup-cron

```diff
@@ -0,0 +1 @@
+* * * * * /usr/local/bin/python /webapp/manage.py runcrons >/dev/null 2>&1
```

Comment:
So although this cron job runs every min the django job only runs once every 12 hours right?

The S3 API costs circa $0.005 per PUT / POST, that's $7.20 per day in requests if its runs every minute... Once every 12 hours or even 24 seems okay.

Reply:
Yes, I can confirm that the PUT to S3 will occur every 12 hours so as to generate two backup files and cost $0.01 per day based on your rate.